### PR TITLE
Issue 3

### PR DESCRIPTION
**Pull request by andreaceccanti**

### CODE REVIEW

File: ci/assets/fetch-rses-from-cric.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+CRIC_URL=${CRIC_URL:-https://escape-cric.cern.ch/api/doma/rse/query/?json&preset=doma}
+VAR_FILE=${VAR_FILE:-test/variables.yaml}
+
+cric=$(curl -s ${CRIC_URL})
```
Comment (andreaceccanti) [inline ci/assets/fetch-rses-from-cric.sh:6]: cric -> rse_list

File: ci/assets/fetch-rses-from-cric.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+for k in "${keys[@]}"; do 
+	typeset -l name=$(echo $cric | jq -r .rses.$k.name)
+	
+	cat ${VAR_FILE} | shyaml keys endpoints.$name > /dev/null 2>&1
+
+	if [ $? -ne 0 ]; then
+		desc=$(echo $cric | jq -r .rses.$k.protocols[0].name)
+		hostname=$(echo $cric | jq -r .rses.$k.protocols[0].hostname)
+		prot=$(echo $cric | jq -r .rses.$k.protocols[0].scheme)
```
Comment (andreaceccanti) [inline ci/assets/fetch-rses-from-cric.sh:19]: prot -> scheme

File: ci/assets/fetch-rses-from-cric.sh (hunk 1 above)
Comment (federicaagostini) [reply ci/assets/fetch-rses-from-cric.sh]: done

File: ci/assets/fetch-rses-from-cric.sh (hunk 2 above)
Comment (federicaagostini) [reply ci/assets/fetch-rses-from-cric.sh]: done

File: .jenkins/Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+pipeline {
+  agent { label 'docker' }
+
+  options {
+    timeout(time: 1, unit: 'HOURS')
+    buildDiscarder(logRotator(numToKeepStr: '5'))
+    ansiColor('xterm')
+    disableConcurrentBuilds()
+    timestamps()
+    lock resource: 'escape-authz-tests-lock'
```
Comment (andreaceccanti) [inline .jenkins/Jenkinsfile:11]: escape-authz-tests-lock -> escape-auth-tests-lock

**Pull request merged**